app/src/main/java/com/besome/sketch/lib/base/BaseWidget.java

```diff
@@ -43,18 +43,18 @@ public int getWidgetType() {
         return d;
     }
 
-    public void setWidgetImage(int n) {
-        c = n;
-        a.setImageResource(n);
+    public void setWidgetImage(int widget_img_res_id) {
+        c = widget_img_res_id;
+        a.setImageResource(widget_img_res_id);
     }
 
-    public void setWidgetName(String string) {
-        b.setText(string);
+    public void setWidgetName(String widgetName) {
+        b.setText(widgetName);
```

Comment:
mixing snake_case and camelCase like that isn't a good idea. normally you should stick to one of them and stay consistent, but in Sketchware Pro's case, the codebase mostly uses camelCase so next time please use that

Reply:
This is only variable and not use in other classes changing it will not affect the other classes.
So it can be changed again

Reply:
of course, but it's still not a good thing

Reply:
Just convert it into widgetImgResId

Reply:
?

Reply:
Oh,No difference.
I earlier single commit, without last commit it didn't worked

Reply:
i just wanted to let you know. i'm gonna change a few things in the classes anyway